.github/workflows/ci-cd.yml

```diff
@@ -92,22 +92,17 @@ jobs:
         uses: actions/setup-python@83679a892e2d95755f2dac6acb0bfd1e9ac5d548
         with:
           allow-prereleases: true
-          cache: pip
-          cache-dependency-path: |
-            requirements/runtime.txt
-            requirements/tests.txt
           check-latest: true
           python-version: ${{ matrix.python-version }}
 
+      - name: Install uv
+        uses: astral-sh/setup-uv@1e862dfacbd1d6d858c55d9b792c756523627244
```

Comment:
Would `pip install uv` be enough here?

Reply:
From what I understand - after checking @KJhellico 's earlier attempt (see https://github.com/KJhellico/python-holidays/commit/adb368b6e6b4760d18d87ca92141a8321d163615), `astral-sh/setup-uv` is supposed to replaces our existing `actions/setup-python` altogether